app/pages/lab/visibility.cjsx

```diff
@@ -211,4 +211,5 @@ module.exports = React.createClass
           </div>
       }</PromiseRenderer>
       <p className="form-help">In a live project active workflows are available to volunteers and cannot be edited. Inactive workflows can be edited if a project is live or in development.</p>
+      <p className="form-help">On a live project, if you want to switch which subjects sets are associated with an active workflow: set the workflow to inactive, next change which subject sets are linked in the Workflow Section within the Project Builder, then return the workflow to active.</p>
```

Comment:
I'd be in favour of just allowing the linking components to be active instead of this faff. The api will allow these changes to a live workflow.